Add term View (#5658)
jenkins-infra/jenkins.io: content/doc/book/glossary/index.adoc

Upstream:: [[upstream]]
    A configured <<pipeline,Pipeline>> or <<job,job>> which triggers a
    separate Pipeline or Job as part of its execution.

> View:: [[view]]
>     A way of displaying data of Jenkins in a dashboard style. Is an extendable object, 
>     so that there are a lot of different ways of e.g. listing <<job,jobs>>, showing trends, analysing data.

Reviewer: ```suggestion
View:: [[view]]
    A way of displaying the data of Jenkins in a dashboard style.
    This is an extensible object, so there are lots of different ways to list <<job,jobs>>, show trends, and analyze data.
```
Small update to make the text read better/smoothly
